shell/platform/common/cpp/client_wrapper/include/flutter/basic_message_channel.h

```diff
@@ -52,8 +52,12 @@ class BasicMessageChannel {
     messenger_->Send(name_, raw_message->data(), raw_message->size());
```

Comment:
As with my comment on the macOS version, I'd much prefer this be changed to call through to the new method with a null reply.

Reply:
Done

Reply:
I found that this might not be the right place to do that. It would make  BinaryMessengerImpl::Send(channel, message, message_size) and FlutterDesktopMessengerSend useless. I could remove them, but didn't want to make too many changes in this PR. I you think it makes sense to remove them, I can do so in a followup PR. 

I did however, change the implementation of FlutterDesktopMessengerSend to call FlutterDesktopMessengerSendWithReply.